projects/aca-shared/src/lib/shared.module.ts

```diff
@@ -28,10 +28,15 @@ import { ContentApiService } from './services/content-api.service';
 import { NodePermissionService } from './services/node-permission.service';
 import { AppService } from './services/app.service';
 import { ContextActionsModule } from './directives/contextmenu/contextmenu.module';
+import { OpenInAppComponent } from './components/open-in-app/open-in-app.component';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatDialogModule } from '@angular/material/dialog';
 
 @NgModule({
-  imports: [ContextActionsModule],
-  exports: [ContextActionsModule]
+  imports: [ContextActionsModule, MatButtonModule, MatIconModule, MatDialogModule],
```

Comment:
Same here, add imports and exports into new lines

Reply:
@MichalKinas prettier formatter sometimes puts those into a single line, but other than that, this PR is missing the linting

Reply:
yess prettier is converting to same line.